Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+properties([])
+node(){
+  deleteDir()
+  stage("Prepare"){
+    checkout scm
+    lint_container = docker.build 'lint'
+  }
+  lint_container.inside {
```

Comment:
`ERROR: The docker version is less than v1.8. Running a 'docker.inside' from inside a container will not work.`

Noticed that in the log, not sure if we're already inside the container and that `.inside` isn't necessary?